aws-kinesis-video-streams/build.gradle

```diff
@@ -0,0 +1,42 @@
+plugins {
+    id 'java'
+    id 'application'
+}
+
+group 'org.example'
+version '1.0-SNAPSHOT'
+
+sourceCompatibility = 1.8
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    testCompile group: 'junit', name: 'junit', version: '4.12'
+
+    implementation "commons-cli:commons-cli:1.4"
+    implementation "org.apache.logging.log4j:log4j-slf4j-impl:2.12.1"
+    implementation "com.google.code.gson:gson:2.8.5"
+    implementation "ai.djl:api:0.3.0"
+    implementation "ai.djl:basicdataset:0.3.0"
+    implementation "ai.djl:model-zoo:0.3.0"
+    implementation "ai.djl.mxnet:mxnet-model-zoo:0.3.0"
+
+    // See https://github.com/awslabs/djl/blob/master/mxnet/mxnet-engine/README.md for more MXNet library selection options
+    runtimeOnly "ai.djl.mxnet:mxnet-native-auto:1.6.0"
+
+    implementation "software.amazon.awssdk:kinesisvideo:2.10.75"
+    implementation "software.amazon.kinesis:amazon-kinesis-client:2.2.9"
+    implementation "com.amazonaws:amazon-kinesis-video-streams-parser-library:1.0.13"
+}
+
+application {
+    mainClassName = System.getProperty("main", "org.example.App")
+}
+
+run {
+    systemProperties System.getProperties()
+    systemProperties.remove("user.dir")
+    systemProperty("file.encoding", "UTF-8")
+}
```

Comment:
configure(this) {
    apply from: file("${rootProject.projectDir}/../tools/gradle/formatter.gradle")
}